tls/vibe/stream/openssl.d

```diff
@@ -849,6 +853,34 @@ final class OpenSSLContext : TLSContext {
 	/// Get the current ALPN callback function
 	@property TLSALPNCallback alpnCallback() const { return m_alpnCallback; }
 
+	/// Callback function which logs the key file (if desired)
+	@property void keylogCallback(SSL_CTX_keylog_cb_func cb)
+	{
+		logDebug("Keylog callback");
+		static if (haveKeylog) {
```

Comment:
I would conditionally define the function, not have it do nothing. Also it would be good to have it have the same behavior as cURL, which is to pick up the environment variable by default.